### PR TITLE
Remove ResponseResult from UpgradeTimeout response, channel-id from UpgradeTry response.

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

closes: #XXXX

### Commit Message / Changelog Entry

```text
chore: remove resp type from UpgradeTimeout response, channel-id from TryUpgrade response.
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: proto/ibc/core/channel/v1/tx.proto [hunk 1]
```diff
@@ -322,7 +322,6 @@ message MsgChannelUpgradeTry {
 message MsgChannelUpgradeTryResponse {
   option (gogoproto.goproto_getters) = false;
 
-  string             channel_id       = 1;
   Upgrade            upgrade          = 2 [(gogoproto.nullable) = false];
   uint64             upgrade_sequence = 3;
   ResponseResultType result           = 4;
```
Comment (colin-axner) [inline proto/ibc/core/channel/v1/tx.proto:327]: need to adjust proto numbers as well

**Review by colin-axner — COMMENTED** [review]
ACK pending review comment

**Review by crodriguezvega — APPROVED** [review]

**Review by damiannolan — APPROVED** [review]
lgtm

**Review by colin-axner — APPROVED** [review]
Thought I had clicked approve yesterday haha

**Pull request merged**